### PR TITLE
Review this to get started

### PR DESCRIPTION
**Pull request by nee1**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/nee1/ReviewNinja-Welcome/pull/1" target="_blank">ReviewNinja

File: ReadMe.md [hunk 1]
```diff
@@ -1 +1,41 @@
-# ReviewNinja-Welcome
\ No newline at end of file
```
Comment (nee1) [inline ReadMe.md]: hey

Comment (nee1) [inline ReadMe.md]: !fixed

Comment (nee1) [inline ReadMe.md]: !fix

**Pull request merged**